Webserver

Denpeer opened the pull request:
Added webserver and python environment.

Denpeer committed:
Python webserver with Flask added, Python virtual environment added

Denpeer committed:
Small homepage update

olivier-dikken committed:
pip install modules into virtualenv

olivier-dikken reviewed (changes requested):
the env doesnt work as expected, looking into just making a requirements.txt file instead

Denpeer committed:
Removed python virtual environment. Requirement list now in repo.
To deploy use virtualenv --no-site-packages --distribute .env && source .env/bin/activate && pip install -r requirements.txt

JoeHarrison reviewed (approved):
Tested the webserver on a virtual environment. All looks good
